<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlottingDialog3</class>
 <widget class="QDialog" name="PlottingDialog3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>双对数曲线设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 统一样式 */
QDialog, QWidget { background-color: white;
color: black; font-family: &quot;Microsoft YaHei&quot;; }
QLabel { color: black; }
QLineEdit, QComboBox, QSpinBox, QDoubleSpinBox { color: black; background-color: white;
border: 1px solid #555; padding: 2px; border-radius: 3px; }
QGroupBox { border: 1px solid #999; margin-top: 10px; padding-top: 10px; border-radius: 3px;
}
QGroupBox::title { subcontrol-origin: margin; left: 10px; padding: 0 3px; color: black; font-weight: bold; }
QPushButton { color: black; background-color: #f0f0f0;
border: 1px solid #555; padding: 5px; border-radius: 3px; }
QCheckBox, QRadioButton { color: black; spacing: 5px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupSource">
     <property name="title">
      <string>数据来源</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutSource">
      <item>
       <widget class="QLabel" name="labelSource">
        <property name="text">
         <string>选择文件:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboFileSelect"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBasic">
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>曲线名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>压差图例:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="linePressLegend"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>导数图例:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineDerivLegend"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupData">
     <property name="title">
      <string>数据与计算</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>时间列:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboTime"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>压力列:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboPress"/>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_Type">
        <item>
         <widget class="QRadioButton" name="radioDrawdown">
          <property name="text">
           <string>压力降落(Drawdown)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBuildup">
          <property name="text">
           <string>压力恢复(Buildup)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPi">
        <property name="text">
         <string>初始压力(MPa):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinPi">
        <property name="maximum">
         <double>9999.990000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_L">
        <property name="text">
         <string>L-Spacing:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spinL">
        <property name="value">
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkSmooth">
          <property name="text">
           <string>启用平滑:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSmooth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAxis">
     <property name="title">
      <string>坐标轴标签</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_X">
        <property name="text">
         <string>X轴:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineXLabel"/>
      </item>
      <item>
       <widget class="QLabel" name="label_Y">
        <property name="text">
         <string>Y轴:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineYLabel"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupStyle">
     <property name="title">
      <string>样式设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_P1">
        <property name="text">
         <string>压差点形:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboPressShape"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_P2">
        <property name="text">
         <string>点色:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnPressPointColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_P3">
        <property name="text">
         <string>压差线型:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboPressLine"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_P4">
        <property name="text">
         <string>线色:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnPressLineColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_D1">
        <property name="text">
         <string>导数点形:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboDerivShape"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_D2">
        <property name="text">
         <string>点色:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnDerivPointColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_D3">
        <property name="text">
         <string>导数线型:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboDerivLine"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_D4">
        <property name="text">
         <string>线色:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnDerivLineColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkNewWindow">
     <property name="text">
      <string>在新建窗口中显示</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlottingDialog3</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlottingDialog3</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
